frontend/src/pages/Broadcast/Component/Carousel/Carousel.jsx

```diff
@@ -62,23 +62,31 @@ export function Carousel(props) {
   const cardImageArrayLight = dataa.map((item, i) => {
     const style = {
       height: "13em",
-      backgroundSize: "cover",
+      backgroundSize: "23.9em",
+      //backgroundSize: "cover", // Cover the entire area
+      margin: "0 auto",
+      backgroundPosition: "center",
+      backgroundRepeat: "no-repeat",
       backgroundBlendMode: "screen",
-      clipPath: "polygon(0 0, 100% 0, 100% 70%, 50% 100%, 0 70%)",
-      backgroundImage: `linear-gradient(45deg,rgba(255, 0, 90, 1) 0%,rgba(10, 24, 61, 1) 90%),url(${item.link})`,
+      // clipPath: "polygon(0 0, 100% 0, 100% 70%, 50% 100%, 0 70%)",
+      backgroundImage: `linear-gradient(45deg,rgba(255, 0, 90, 1) 0%,rgba(10, 24, 61, 1) 90%),url(${item.imageUrl[0]})`,
     };
     return style;
   });
 
   const cardImageArrayDark = dataa.map((item, i) => {
     const style = {
       height: "13em",
-      backgroundSize: "cover",
+      backgroundSize: "23.9em",
+      //backgroundSize: "cover", // Cover the entire area
```

Comment:
remove all commented code and i would prefer the image to cover the whole space instead of fixed height